.travis.yml

```diff
@@ -0,0 +1,8 @@
+language: node_js
+node_js:
+  - "8"
+install:
+  - npm install
+  - npm run test
+script:
+  - npm run build -s
```

Comment:
Please add `npm run test:lint` here.

Reply:
I added headless chrome according to @saadataz comment.